src/runtime/vdom/vdom-render.ts

```diff
@@ -683,50 +688,59 @@ const updateFallbackSlotVisibility = (elm: d.RenderNode) => {
   }
 };
 
+/**
+ * Component-global information about nodes which are either currently being
+ * relocated or will be shortly.
+ */
 const relocateNodes: RelocateNodeData[] = [];
 
+/**
+ * Relocate the content of a slot
+ *
+ * @param elm a render node whose child nodes need to be relocated
+ */
```

Comment:
This method is one that might benefit from a rename. It's a bit confusing since the relocation doesn't actually happen here. We just figure out which nodes should be moved and where they'll be moved to (if anywhere). It kinda just generates a map from an original location to a new location.

Reply:
right because the nodes get added to the `relocateNodes` data structure...I'm with you on the renaming, how about `markSlotContentForRelocation`? I'll add a node to the jsdoc clarifying that this doesn't actually move the nodes but just puts them into the 'queue' for that to happen